.github/workflows/build.yml

```diff
@@ -4,6 +4,8 @@ on:
   push:
     branches:
       - main
+    paths-ignore:
+      - "docs/**"
```

Comment:
Do we also need to mark the `.github` actions like in `test.yml`?

Reply:
We don't need to, but I will add some of the others, the main thing we want to avoid is preventing rebuilds when the citation changes as we have to rebuild following steps 5 and 6 in https://github.com/muon-spectroscopy-computational-project/Developer-notes/blob/main/package-maintenance.md.